port/unix/omrsysinfo.c

```diff
@@ -3431,6 +3472,7 @@ readCgroupFile(struct OMRPortLibrary *portLibrary, int pid, OMRCgroupEntry **cgr
 	requiredSize = portLibrary->str_printf(portLibrary, NULL, (uint32_t)-1, "/proc/%d/cgroup", pid);
 	Assert_PRT_true(requiredSize <= PATH_MAX);
 	portLibrary->str_printf(portLibrary, cgroup, sizeof(cgroup), "/proc/%d/cgroup", pid);
+
```

Comment:
`cgroup` at this point is storing path `/proc/pid/cgroup`. Same var is also used for storing cgroup name by parsing `/proc/pid/cgroup` file. This may be the cause of confusion. I will update code to rename the variable.
Use of `ROOT_CGROUP` is intentional as we don't want to use the cgroup name we get from `/proc/pid/cgroup` file.